adlfs/multithread.py

```diff
@@ -183,6 +185,7 @@ def get_chunk(adlfs, src, dst, offset, size, blocksize, retries=MAXRETRIES,
                             logger.debug('Aborting %s, byte offset %s', dst, o)
                             raise
```

Comment:
Return `False`? I thought we wanted to avoid passing back the exception, which might be keeping the stack live following an exception. That would need a check for `False` at the far end, rather than checking the error status.